packages/web/modals/validator-squad.tsx

```diff
@@ -487,15 +510,88 @@ export const ValidatorSquadModal: FunctionComponent<ValidatorSquadModalProps> =
             className="max-h-[33rem] overflow-y-scroll md:max-h-[18.75rem]" // 528px & md:300px
             ref={tableContainerRef}
           >
-            <ValidatorSquadTable
-              // @ts-ignore
-              sorting={sorting}
-              setSorting={setSorting}
-              filteredValidators={filteredValidators}
-              setRowSelection={setRowSelection}
-              rowSelection={rowSelection}
-              table={table}
-            />
+            <table className="w-full border-separate border-spacing-y-1">
```

Comment:
why the change from having this extracted?

Reply:
Interesting. I ran into something similar in my refactor of the sidebar. It was specific to NextJS. I literally just extracted the component and it caused a mismatch between SSR and client side render. I was able to figure it out eventually but it took a few hours of research.